src/Illuminate/Http/Resources/DelegatesToResource.php

```diff
@@ -46,7 +46,7 @@ public function resolveRouteBinding($value)
      */
     public function offsetExists($offset)
     {
-        return array_key_exists($this->resource[$offset]);
+        return array_key_exists($this->resource, $offset);
```

Comment:
Also I believe you mean:

```
return array_key_exists($offset, $this->resource);
```

Reply:
Good catch, fixed: https://github.com/laravel/framework/pull/20887/commits/c907481d94bc9d9c80f3c5af1f6d63add23d1a31

Reply:
In all seriousness - why did the tests not catch that error?